### PR TITLE
Cleanup in ethereum connector service

### PR DESCRIPTION
**Pull request by Ram-srini**
Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: listener/avalon_listener/tcs_work_order_handler.py [hunk 1]
```diff
@@ -206,6 +206,11 @@ def WorkOrderSubmit(self, **params):
               as defined in EEA spec 6.1.1
         Returns jrpc response as defined in EEA spec 6.1.3
         """
+        if "workOrderId" not in params:
+            raise JSONRPCDispatchException(
+                WorkOrderStatus.INVALID_PARAMETER_FORMAT_OR_VALUE,
+                "Missing work order id"
+            )
```
Comment (manojgop) [inline listener/avalon_listener/tcs_work_order_handler.py:213]: Need a similar check in https://github.com/hyperledger/avalon/blob/master/listener/avalon_listener/tcs_work_order_handler_sync.py#L65. 
But rather we should have a common function for both files

Comment (Ram-srini) [reply listener/avalon_listener/tcs_work_order_handler.py:213]: This could be fixed using jsonschema as well.

**Comment by rranjan3** [comment]
Depends on #600

File: listener/avalon_listener/tcs_work_order_handler.py (hunk 1 above)
Comment (Ram-srini) [reply listener/avalon_listener/tcs_work_order_handler.py]: @manojgop Removed since json-schema address it.

**Pull request merged**